database/factories/DoctorFactory.php

```diff
@@ -25,9 +25,12 @@ public function definition()
         return [
             'name' => $this->faker->name,
             'email' => $this->faker->unique()->safeEmail,
-            'email_verified_at' => now(),
-            'password' => '$2y$10$92IXUNpkjO0rOQ5byMi.Ye4oKoEa3Ro9llC/.og/at2.uheWG/igi', // password
-            'remember_token' => Str::random(10),
+            'phone' => $this->faker->unique()->phoneNumber,
+            'specialization_id' => 1,
+            'photo' => UploadedFile::fake()->image('photo.jpg'),
+            'degree_copy' => UploadedFile::fake()->image('degree.jpg'),
+            'gender' => '1',
```

Comment:
Do not use hardcoded values for enum attributes
Place it with GenderType::MALE

Reply:
done